src/clusterfuzz/_internal/datastore/data_types.py

```diff
@@ -1327,6 +1327,12 @@ class FiledBug(Model):
   # Platform id.
   platform_id = ndb.StringProperty()
 
+  # Project name that is associated with the filed issue.
+  project_name = ndb.StringProperty()
```

Comment:
You'll likely need to update to add a query index for this here: https://github.com/google/clusterfuzz/blob/992274ebd11e178b2c664ec2fa684ff7d98df376/src/appengine/index.yaml#L4